src/components/app-wrapper/app-cached-data-query-provider.tsx

```diff
@@ -109,6 +110,9 @@ const providerDataTransformation = ({
             uiLocale: currentUser.settings?.keyUiLocale ?? 'en',
             displayProperty: currentUser.settings?.keyAnalysisDisplayProperty,
             displayNameProperty,
+            // preserve this original key used in the InterpretationModal component in analytics
+            keyAnalysisDisplayProperty:
+                currentUser.settings?.keyAnalysisDisplayProperty,
```

Comment:
Perhaps it would be nicer if we actually just address this in the InterpretationsProvider?

Reply:
I think it makes sense.
I also don't like to have to add this "duplicated" key in `currentUser`.